### DISCUSSION TITLE
Where is the wiki config file located and how can I access it?

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6350] [Help / Questions]
Where can I adjust the upload size limit of pages? I know where the setting is in the administration area, but changes there do not seem to apply. So where is the config file in the wiki container and how can I access it and change an entry? 

Is that even the right location to search for it? 

Thanks for any input.

Version: 2.5.294
Docker container on ubuntu 20.04 (image installation with wiki.js documentation)

**Comment by fnafox** [comment]
In docker setups the configuration is set with environment variables, alternatively you can mount the file externally. Check the [documentation](https://docs.requarks.io/install/docker#alternative-mount-the-config-file).

However if I recall correctly that value is not adjusted in the config, the option is in the Admin > Security > Uploads section.
If you're using a reverse proxy make sure that your post_max_size and upload_max_filesize variables on the proxy webserver are set high enough as well.

**Reply by ghost** [reply]
Thanks for your reply. 

Unfortunately I tried to change the option on that path Admin > Security > Uploads but it seems like it did not apply the changes on another layer, because my problem still persists. Also, I am not using any reverse proxy. I have installed wiki.js via docker image on an ubuntu virtual machine like in the wiki.js documentation: https://docs.requarks.io/install/ubuntu

So its quite hard for me to understand where I should look for that problem.
